### PR TITLE
Show clipped description in simple list for IE11

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 
## Issue
https://github.com/Fliplet/fliplet-studio/issues/5030

## Description
Show clipped description in a simple list for IE11

## Screenshots/screencasts
MS IE11
image

MS EDGE
image

Firefox
image

Chrome
image

## Backward compatibility

This change is fully backward compatible.

## Notes
Use set interval function to wait until the element will appear in the DOM.

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -7,4 +7,19 @@ $('[data-dynamic-lists-id]').each(function(){
   if (!data.layout) {
     dynamicLists[id] = new DynamicLists(data, container);
   }
-});
+
+  if (data.layout === 'simple-list' || data.layout === 'news-feed') {
+    var showLines = Modernizr.ie11 ? 4 : 3;
+    var descriptionSelector = data.layout === 'simple-list' ? '.list-item-description' : '.news-feed-item-description';
```
Comment (tonytlwu) [inline js/build.js]:
```suggestion
    var descriptionSelector = data.layout === 'simple-list'
      ? '.list-item-description'
      : '.news-feed-item-description';
```

File: js/build.js [hunk 2]
```diff
@@ -7,4 +7,19 @@ $('[data-dynamic-lists-id]').each(function(){
...
+    var awaitForDom = setInterval(function() {
+      if ($(descriptionSelector).length) {
+        clearInterval(awaitForDom);
+        var descriptions = document.querySelectorAll(descriptionSelector);
+        for (var i = 0; i < descriptions.length; i++) {
```
Comment (tonytlwu) [inline js/build.js]:
```suggestion

        for (var i = 0; i < descriptions.length; i++) {
```

**Comment by YaroslavOvdii** [comment]
@tonytlwu Updated code with your suggestions.

File: js/build.js [hunk 3]
```diff
@@ -7,4 +7,22 @@ $('[data-dynamic-lists-id]').each(function(){
...
+
+  if (data.layout === 'simple-list' || data.layout === 'news-feed') {
+    var showLines = Modernizr.ie11 ? 4 : 3;
+    var descriptionSelector = data.layout === 'simple-list'
+      ? '.list-item-description'
+      : '.news-feed-item-description';
+    var awaitForDom = setInterval(function() {
+      if ($(descriptionSelector).length) {
+        clearInterval(awaitForDom);
+        var descriptions = document.querySelectorAll(descriptionSelector);
```
Comment (tonytlwu) [inline js/build.js]:
```suggestion

        var descriptions = document.querySelectorAll(descriptionSelector);
```

**Comment by YaroslavOvdii** [comment]
@tonytlwu Updated

**Pull request merged**